README.md

```diff
@@ -183,14 +183,14 @@ Result is an object:
 
 ##### async
 
-Type: `Boolean`  
```

Comment:
Please, revert spaces at the end of line – it's a feature of markdown. Otherwise it looks like
![image](https://user-images.githubusercontent.com/270491/101211530-9c867e00-3677-11eb-9440-eb287d54163a.png)
instead of
![image](https://user-images.githubusercontent.com/270491/101211626-bfb12d80-3677-11eb-984a-41b0c712586c.png)

Reply:
WebStorm did it. Sorry. Just 1 minute =)